README.md

```diff
@@ -2,76 +2,27 @@
 Bowling Challenge
 =================
 
+## SETUP 
 
-* Challenge time: rest of the day and weekend.
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday week
+* Clone the project into a folder of your choice
+* Run 'npm install' to install relevant npm packages
```

Comment:
In fact, I found I had to run npm install in two places: the root directory, and then also in client. When I had only run npm install in the root directory, I got:

```
$ cd client
$ npm start

> client@0.1.0 start /Users/david/makers_apprenticeships/assessments/bowling-challenge/client
> react-scripts start

sh: react-scripts: command not found
```

I notice you have package.jsons at two levels: in the 'client' folder and in the root directory. So I now have two node_modules folders. Ideally these dependencies would be all managed in one place, as I'm sure you know. And I see there are two copies of 'bowling.js', one for the client and one for the tests; again, it would be good to only need to change one file to change one thing.